packages/language-service/src/lib/provider.ts

```diff
@@ -182,26 +197,33 @@ export class Provider {
                     break;
                 }
                 case 'import': {
-                    let rslvd: CSSResolve | JSResolve | null = null;
+                    let resolved: CSSResolve | JSResolve | null = null;
                     try {
-                        rslvd = this.stylable.resolver.resolve(symb);
+                        resolved = this.stylable.resolver.resolve(symbol);
                     } catch {
                         /**/
                     }
 
-                    let filePath: string;
+                    let filePath;
```

Comment:
It will always be `any`

Reply:
why not set it as `let filePath = ""`?

Reply:
fixed it to be `string | undefined` which also works.